javascript-loops/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+getNumbersToTen();
```

Comment:
You don't need to call/invoke functions in this file — please remove all the function calls